app/src/main/res/layout/post_match.xml

```diff
@@ -6,15 +6,15 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/cpr_bkgnd"
-    tools:context=".PostMatch">
+    tools:context=".PostMatch"
+    tools:ignore="VisualLintBounds">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:orientation="vertical"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
```

Comment:
What changed here and why?

Reply:
I don't think I changed anything, it must have been an accident. Do you want it changed or is it fine?